### DISCUSSION TITLE
Fluent v9 React and Fluent Design System 2

### DISCUSSION THREAD

**Discussion by shawnkelshaw** [microsoft/fluentui#29609] [General] [answered]
Before I get started on a new project, I wanted to make sure that these two resources are related.

We are planning to use:
[Fluent React v9.35.1](https://react.fluentui.dev/?path=/docs/concepts-introduction--page) as our component UI stack.

Is the [Fluent 2 Design System](https://fluent2.microsoft.design/get-started/whatisnew) related and appropriate to leverage? It is unclear if these two repositories are related. I think they are, but wanted to double check since I can't find an explicit statement confirming.

Thanks.

**Comment by khmakoto** [comment, marked as answer]
Hi @shawnkelshaw, yes, the Fluent UI React v9 library is an implementation in React of the Fluent 2 design system, so you should be good to use it on your project. 😄

**Reply by shawnkelshaw** [reply]
Thank you. Additionally, I like your name. It is fun to say out loud. :)

**Reply by khmakoto** [reply]
Thanks! 😄

**Comment by danmarshall** [comment]
thanks for the answer @shawnkelshaw . I found the versioning/numbering/naming confusing. It would also be helpful if the Fluent2 website had links back to the GitHub (for page edits, etc.)
